x-pack/legacy/server/lib/mirror_plugin_status.js

```diff
@@ -5,7 +5,7 @@
  */
 
 export function mirrorPluginStatus(upstreamPlugin, downstreamPlugin, ...statesToMirror) {
-  upstreamPlugin.status.setMaxListeners(20); // We need more than the default, which is 10
+  upstreamPlugin.status.setMaxListeners(30); // We need more than the default, which is 10
```

Comment:
I have no context here, so I'm curious what necessitated this change and whether it impacts other teams?

Reply:
So it seems that we reached the 20 limits and my plugin was going to 21. I changed it to 30 and everything worked fine. You are right that we might want to ask what impact this could have on Kibana in general.